plugins/login-resources/src/utils.ts

```diff
@@ -67,6 +67,9 @@ export async function doLogin (email: string, password: string): Promise<[Status
       },
       body: JSON.stringify(request)
     })
+    if (!emailValidator(email)) {
```

Comment:
this logic contains the same as previous, I just observe that logIn function needs to be updated as signUp too.